tests/unittests/server/routes/test_camera.py

```
@@ -0,0 +1,765 @@
+
+class TestBatchCameraIntrinsicsRoute:
+    """Tests for BatchCameraIntrinsics endpoint."""
+
+    @pytest.fixture(name="multi_sample_dataset")
+    def fixture_multi_sample_dataset(self):
+        """Creates a dataset with multiple samples."""
+        dataset = fo.Dataset()
+        dataset.persistent = True
+
+        samples = [
+            fo.Sample(filepath=f"/tmp/test_camera_{i}.jpg") for i in range(3)
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/test_camera_"

(S108)